ingestion-core/src/metadata/_version.py

```diff
@@ -0,0 +1,11 @@
+"""
+Provides metadata version information.
+"""
+
+# This file is auto-generated! Do not edit!
+# Use `python -m incremental.update metadata` to change this file.
+
+from incremental import Version
+
+__version__ = Version("metadata", 0, 6, 0, dev=2)
```

Comment:
let's change this to 0.8.0. I want the versioning of openMetadata backend and ingestion to track each other

Reply:
Thanks Harsha. What should be the version string to update ? `0.8.0.dev0` ?